apps/prairielearn/src/lib/question-render.ts

```diff
@@ -15,12 +15,12 @@ import {
 } from '../components/QuestionContainer.types.js';
 import { QuestionNavSideButton } from '../components/QuestionNavigation.html.js';
 import { QuestionScorePanelContent } from '../components/QuestionScore.html.js';
+import type { SubmissionForRender } from '../components/SubmissionPanel.html.js';
 import {
   SubmissionBasicSchema,
   SubmissionDetailedSchema,
   SubmissionPanel,
 } from '../components/SubmissionPanel.html.js';
```

Comment:
Nit: combine these imports?
```suggestion
import {
  SubmissionBasicSchema,
  SubmissionDetailedSchema,
  type SubmissionForRender,
  SubmissionPanel,
} from '../components/SubmissionPanel.html.js';
```